php/Terminus/Collections/Instruments.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Terminus\Collections;
+use Terminus\Request;
+use Terminus\User;
+use Terminus\Instrument;
+use \Terminus_Command;
+
+class Instruments {
+  private $_user;
+  private $_models = array();
+
+  public function __construct($user = false, $options = array()) {
```

Comment:
I'd prefer if we passed the User relation in via the `$options` array. I think the first argument of collections should probably be a list of models (to match Backbonejs's collection constructor). But mostly I prefer being very declarative of of passing in things via an associative array versus arguments. 

(I figure you were just copy-pasting from the other collections, whose interface unfortunately isn't as mature as my thoughts about it)